packages/modules/cra-template-itwin-viewer/template/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "iTwin Viewer",
+  "name": "iTwin Viewer React Sample",
```

Comment:
Maybe remove the words React Sample from this since it is the CRA templatre

Reply:
```suggestion
  "name": "iTwin Viewer",
```